src/declarative/activitymodel.cpp

```diff
@@ -352,6 +352,16 @@ QString ActivityModel::currentActivityTime() const
     return QString();
 }
 
+QString ActivityModel::totalActivityTime() const
+{
+    QTime totalTime = QTime(0, 0);
```

Comment:
QTime totalTime;

You don't need to assign this to QTime(0, 0) as implicit contructor already constructs it. I'll have to change my code as well as I use QTime(0, 0) too.

Reply:
ah, good to know! Thanks!